doc/styling/wind-recipe.md

```
@@ -0,0 +1,229 @@
+WindSlotCompoundVariant({
+  required Map<String, Object> conditions,
+  required Map<String, String> classNames,
+})
+```
+
+<a name="resolution-order"></a>
+## Resolution Order
+
+Every call resolves in this strict order: no dedupe, no sort, no twMerge:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)